austation/code/modules/atmospherics/gasmixtures/auxgm.dm

```diff
@@ -0,0 +1,6 @@
+/datum/gas
+	var/custom_icon = null //custom icon filepath for snowflakey gases
+	var/custom_x = null
+	var/custom_y = null
+	var/vis_flags = null
+	var/layers = null
```

Comment:
Wouldn't it be better to make this it's own subtype? Adding all of these (moreso all of checks they add in new) adds unneeded overhead to gas creation

Reply:
there's already a shit ton of vars, a few more won't do shit